<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeModifyUserDlg</class>
 <widget class="QDialog" name="DeModifyUserDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeModifyUserDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit
{
border:1px solid gray;
border-radius:5px;
min-height:30px;
font: 16pt &quot;新宋体&quot;;
}

#widget_2
{
min-height:30px;
}

QLabel
{
font: 14pt &quot;新宋体&quot;;
min-width:60px;
}

QGroupBox {
     border: 1px solid gray;
     border-radius: 3px;
     margin-top: 2ex; /* leave space at the top for the title */
     padding: 10px;
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }

QPushButton {
    
	font: bold 16pt &quot;新宋体&quot;;
	background-color: rgb(244, 244, 244);
     border: 0px solid #8f8f91;
     border-radius: 5px;
     min-width: 100px;
     min-height: 30px;
 }

 QPushButton:pressed {
	background-color: rgb(255, 255, 255);
 }
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="styleSheet">
      <string notr="true">
QPushButton::focus
{
    color: rgb(254, 255, 253);
    background-color: rgb(66, 189, 170);
}

QPushButton::!focus
{    
	color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">max-width: 320px;
max-height:180px;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="4">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>修改用户</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>用户名:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="userLineEdit">
              <property name="maxLength">
               <number>32</number>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>密  码:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="passwdLineEdit">
              <property name="maxLength">
               <number>32</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>权  限:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="DeComboBox" name="widgePermission" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>37</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="okBtn">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="cancelBtn">
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>37</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="DeVirtualKeyboradWidget" name="keyBoradWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DeVirtualKeyboradWidget</class>
   <extends>QWidget</extends>
   <header location="global">devirtualkeyboradwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DeComboBox</class>
   <extends>QWidget</extends>
   <header location="global">decombobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
